### ISSUE TITLE
cfn-lint 0.49.1 does not catch `/` as an invalid character in a Mapping element name

### ISSUE THREAD

**Issue by ps-jay** [aws-cloudformation/cfn-python-lint#2004]
*cfn-lint version: cfn-lint 0.49.1*

*cfn-lint did not catch `/` as an invalid character in a Mapping element name*

cfn-lint passed successfully with this mapping included in the template:
```yaml
Mappings:
  NameServers:
    10.90.0.0/16:
      NameServer1: 10.90.0.10
      NameServer2: 10.90.4.10
    10.91.0.0/16:
      NameServer1: 10.91.0.10
      NameServer2: 10.91.4.10
```

However AWS rejected it:
> Template format error: Mappings element name '10.93.0.0/16' must be non-empty and can contain only alphanumerics, '-' or '.'

![Screen Shot 2021-05-12 at 11 34 41](https://user-images.githubusercontent.com/7205587/117905164-0e917c00-b316-11eb-8132-2da240ff892a.png)

**Comment by PatMyron** [comment]
Thanks for reporting, does seem to be missing that level:

<img width="656" alt="Screen Shot 2021-05-11 at 9 49 34 PM" src="https://user-images.githubusercontent.com/7014355/117906384-ce44e600-b2a2-11eb-9ad7-1d900c5ec4bd.png">

**Comment by kddejong** [comment]
@PatMyron According to the [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/mappings-section-structure.html) `The name can contain only alphanumeric characters (A-Za-z0-9)`  This isn't fully true as you can do regions like `us-east-1` in the key names. Do you know what characters are valid.

We actually disabled the alphanumeric check for this because I couldn't find what a correct regex would be.
https://github.com/aws-cloudformation/cfn-python-lint/blob/853522ff0e4c977331ac677a47059cf96b6f3189/src/cfnlint/rules/mappings/KeyName.py#L40

**Comment by kddejong** [comment]
I think I could convert to `^[a-zA-Z0-9._-]{1,255}$` but not sure that is valid for all levels either.  Going to test a few things.

**Comment by kddejong** [comment]
```yaml
Mappings:
  region-name:
    us-east-1:
      key-1: value
```

`Mappings name 'region-name' is non alphanumeric.`
`Mappings attribute name 'key-1' must contain only alphanumeric characters.`

So it appears just the key where `us-east-1` has a different pattern :/
